Footer is no longer fixed at bottom, instead it is at the very bottom

ctwoolsey opened the pull request.

abobwhite commented on app/styles/main.scss:
```diff
@@ -57,9 +56,12 @@ $footerHeight : 60px;
 footer {
   width: 100%;
   height: $footerHeight;
-  line-height: $footerHeight;
```
Why did you remove line-height? This existed so the content in the footer was vertically centered...

ctwoolsey commented:
Yes, I knew why the line height was there.  I don't remember exactly what
the problem was, but the line-height was causing an extra margin at the
bottom of the page or some odd behavior which was fixed by removing it.

On Thu, Nov 10, 2016 at 7:59 AM, Alex White <notifications@github.com>
wrote:

> *@abobwhite* requested changes on this pull request.
> ------------------------------
>
> In app/styles/main.scss
> <https://github.com/OpenDataSTL/STLCourts-client/pull/20#pullrequestreview-8011844>
> :
>
> > @@ -57,9 +56,12 @@ $footerHeight : 60px;
>  footer {
>    width: 100%;
>    height: $footerHeight;
> -  line-height: $footerHeight;
>
> Why did you remove line-height? This existed so the content in the footer
> was vertically centered...
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/OpenDataSTL/STLCourts-client/pull/20#pullrequestreview-8011844>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AP0KxjS-f99kcm9_3TIOwr47GFgzLHJ8ks5q8yM-gaJpZM4Knf_l>
> .
>

ctwoolsey commented:
[image: Inline image 1]
This is what was happening with the line-height set.  Removing line-height
fixed this and so I didn't bother looking for another solution.

On Thu, Nov 10, 2016 at 8:27 AM, Clark Woolsey <clark.woolsey@gmail.com>
wrote:

> Yes, I knew why the line height was there.  I don't remember exactly what
> the problem was, but the line-height was causing an extra margin at the
> bottom of the page or some odd behavior which was fixed by removing it.
>
> On Thu, Nov 10, 2016 at 7:59 AM, Alex White <notifications@github.com>
> wrote:
>
>> *@abobwhite* requested changes on this pull request.
>> ------------------------------
>>
>> In app/styles/main.scss
>> <https://github.com/OpenDataSTL/STLCourts-client/pull/20#pullrequestreview-8011844>
>> :
>>
>> > @@ -57,9 +56,12 @@ $footerHeight : 60px;
>>  footer {
>>    width: 100%;
>>    height: $footerHeight;
>> -  line-height: $footerHeight;
>>
>> Why did you remove line-height? This existed so the content in the footer
>> was vertically centered...
>>
>> —
>> You are receiving this because you authored the thread.
>> Reply to this email directly, view it on GitHub
>> <https://github.com/OpenDataSTL/STLCourts-client/pull/20#pullrequestreview-8011844>,
>> or mute the thread
>> <https://github.com/notifications/unsubscribe-auth/AP0KxjS-f99kcm9_3TIOwr47GFgzLHJ8ks5q8yM-gaJpZM4Knf_l>
>> .
>>
>
>

ctwoolsey commented:
![image](https://cloud.githubusercontent.com/assets/16583366/20181761/0730fe56-a725-11e6-89d0-292422f643ed.png)

abobwhite commented:
Why also did you remove `display: table-cell`? The combination of your css changes could have made the `line-height` an issue.

Is everything still vertically aligned to the middle?

ctwoolsey commented:
Yes everything is vertically aligned to the middle.  I don't remember why I removed the Table-cell.  I cleaned up the footer code and the table-cell didn't seem to be doing anything if I remember correctly.  Or if it was doing something, it was breaking something else.  But the footer should look nice and clean with the text vertically centered. I added a padding to the top of the footer and the text looks nicely centered and scales.

ctwoolsey commented:
When I make the line height equal to the footer height it has this problem.  I could make the line-height equal to 50px instead of the 60px (the footer height) and then I could remove the 15px padding on the footer-top.  And it still looks good and this mysterious bottom margin then disappears...Or I could leave it like it is because it looks good and works.

ctwoolsey commented:
Here is how it looks currently:
![image](https://cloud.githubusercontent.com/assets/16583366/20185971/c78be3d8-a733-11e6-995a-e136279104e8.png)

ctwoolsey commented:
Actually, Let me look at this again, because I'm seeing some issue when the page is smaller, that I didn't see before.

abobwhite commented:
@ctwoolsey Please merge develop into this branch since it currently has a merge conflict with footer.html

ctwoolsey commented:
OK, this should be ready to go now.

abobwhite commented:
@ctwoolsey This is still showing 13 files changes over 5 commits. You only changed 3 files in your branch.

ctwoolsey commented:
Yes, somehow it did that, but I did a revert on those commits.  So it really only affects the proper files.  Not sure how I can get rid of the other fIles unless I try creating a new branch and delete this one.

abobwhite commented:
Something will have to be done. Because these are not changes you made in this branch and they should not show up here. Maybe a new branch might be the easiest without us looking at it.

abobwhite commented:
Closed in favor of #25
